src/components/tables/Details/StatusIndicatorAndLabel.tsx

```diff
@@ -2,16 +2,18 @@ import { TableCell, Typography, useTheme } from '@mui/material';
 import { Shard } from 'data-plane-gateway/types/shard_client';
 import { FormattedMessage } from 'react-intl';
 import {
-    ShardStatusColor,
     useShardDetail_evaluateShardProcessingState,
     useShardDetail_getShardStatusColor,
     useShardDetail_getShardStatusMessageId,
-} from 'stores/ShardDetail';
+} from 'stores/ShardDetail/hooks';
+import { ShardStatusColor } from 'stores/ShardDetail/types';
 
 interface Props {
     shard: Shard;
 }
 
+export const PREFIX_NAME_PATTERN = `[a-zA-Z0-9-_.]+`;
```

Comment:
Why is this constant defined and exported from this file? I see that it is used within the details form (i.e., _src/components/shared/Entity/DetailsForm/Form.tsx_) but not here, unless I am mistaken.

Reply:
No clue... I think I pasted this here planning to move it to another file... but that's a total guess as this is does not belong here in anyway at all.

Reply:
moved to mist-utils